docs: Fix misspells in Ranker and List docs

makseq opened the pull request:
#### Change has impacts in these area(s)
- [x] Frontend docs



### Describe the reason for change
Fix misspells in Ranker and List tags docs. 


#### What does this fix?
Excess commas and adding a new comment line

makseq committed:
docs: Fix misspells in Ranker and List docs

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8446519502

hlomzik commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/8450239349)
> Successfully merged:  16 files changed, 552 insertions(+), 124 deletions(-)

hlomzik committed:
Merge branch 'develop' into 'fix-misspells-frontend-docs'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8450239349

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8450263804

hlomzik commented on web/libs/editor/src/tags/control/Ranker.js:
```diff
@@ -32,14 +32,15 @@ const ORIGINAL_ITEMS_KEY = '_';
+ *     { "id": "wiki", "title": "Arrow (computer science)", "body": "In computer science, arrows or bolts are a type class..." }
  *   ]
  * }
+ * <!-- Example result for Ranker tag -->
```
I think you need to specify it as a new example
```suggestion
 * @example
 * <!-- Example result for Ranker tag -->
```

hlomzik reviewed (changes requested):
-

hlomzik committed:
Add one more `@example` tag

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8450357277
